### DISCUSSION TITLE
Hazine hangi kutuda

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#199] [Zeka Sorusu]
İki kişinin tuttukları yazılı not bulunan iki kutudan birisinde bir hazine vardır. Kutuların üzerindeki notlar aşağıdaki gibidir:
1.kutu: Bu kutu boştur.
2.kutu: Bu kutuların üzerindeki notlardan sadece birisi doğrudur.
Buna göre hazine hangi kutudadır?

**Comment by akgembi12** [comment]
Hazine 1. Kutidadır 
1. Kutunun söylediği yanlış 2. Kutunun söylediği yanlış da doğru da olabilir
